internal/service/httpserver.go

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
```

Comment:
This file copied from 3.0 go-mod-bootstrap before convert of MUX to Echo. Removed the CORS and Middleware calls/code since those use Echo and aren't needed for the UI.